format/Message.fbs

```diff
@@ -93,6 +93,28 @@ table Time {
 /// time from the Unix epoch, 00:00:00.000 on 1 January 1970, UTC.
 table Timestamp {
   unit: TimeUnit;
+
+  /// The time zone is a string indicating the name of a time zone, one of:
+  ///
+  /// * As used in the Olson time zone database (the "tz database" or
+  ///   "tzdata"), such as "America/New_York"
+  /// * An absolute time zone offset of the form +XX:XX or -XX:XX, such as +07:30
```

Comment:
Right: Julian clarified here:
https://issues.apache.org/jira/browse/ARROW-637?focusedCommentId=15928783&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-15928783
Sorry for the duplicated question.